apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-logo/edit.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useEntityProp, __experimentalUseEntitySaving } from '@wordpress/core-data';
+import { IconButton, PanelBody, RangeControl, Toolbar } from '@wordpress/components';
+import {
+	BlockControls,
+	BlockAlignmentToolbar,
+	BlockIcon,
+	InspectorControls,
+	MediaPlaceholder,
+} from '@wordpress/block-editor';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import icon from './icon';
+
+const onError = message => {
+	console.log( message );
+};
+
+export default function LogoEdit( {
+	attributes: { align, width },
+	children,
+	className,
+	setAttributes,
+} ) {
+	const [ isEditing, setIsEditing ] = useState( false );
+	const [ logo, setLogo ] = useEntityProp( 'root', 'site', 'sitelogo' );
+	const [ isDirty, , save ] = __experimentalUseEntitySaving( 'root', 'site', 'sitelogo' );
```

Comment:
When I have the logo block saved in the post content of a template part in FSE (like the header), changing the image does not make the post dirty, so I can never update the logo.

For our existing FSE blocks, we're using a custom HOC to work around this since entity saving did not exist when FSE was created.

Reply:
Ah, this is going to be a problem :(